nginx_proxy/start_nginx.sh

```diff
@@ -32,25 +33,38 @@ $(command basename $0)
 (2) Starts nginx with a custom Endpoints service name and service version, and
 have nginx obtain the service configuration:
 $(command basename $0) -n ENDPOINTS_SERVICE_NAME -v ENDPOINTS_SERVICE_VERSION
+(3) Starts nginx with a custom Endpoints service name and managed rollout
+strategy
+$(command basename $0) -n ENDPOINTS_SERVICE_NAME -r managed
 Options:
     -h
         Shows this message.
     -n ENDPOINTS_SERVICE_NAME
         Required. The name of the Endpoints Service.
```

Comment:
This should be Optional rather than Required.

Reply:
Done